WINDOWS.md

```diff
@@ -19,9 +19,19 @@ If you are running Windows 8+, you need to disable Fast Startup :
 1. Right click -> Download this: [turn_off_fast_startup.bat](https://raw.githubusercontent.com/lewagon/setup/master/utils/turn_off_fast_startup.bat).
 1. Right click on the downloaded file and click **Run as administrator**
 
+**OR**
+
+Go to **Energy Settings** -> **Power Button** -> **More** -> **Disable FastBoot**
+
 ## Install Ubuntu
 
-Plug-in the USB stick, and restart your computer. If it does not boot on Linux, you need to ask a teacher to help you with accessing the BIOS and changing the boot order so that it reads from the usb stick first before the hard drive.
+1. Plug-in the USB stick, and restart your computer: **Modify Reboot Option** -> **Advanced Boot**.
+2. Choose **Removable Device**. If it does not boot on Linux, you need to ask a teacher to help you with accessing the BIOS and changing the boot order so that it reads from the usb stick first before the hard drive ([help](UBUNTU_ISSUES.md)).
+3. Install Ubuntu in **english**, choose **Third-Party** install and alongside with Windows.
```

Comment:
Actually, not not "tick" the third party install + do not "tick" the "Update ..." as it requires an internet connection. It's better to do it without.